.github/workflows/release.yml

```diff
@@ -24,9 +24,12 @@ jobs:
       - name: Install Dependencies
         run: npm install
 
-      - name: Publish to npm
+      - name: Create Release Pull Request or Publish to npm
         uses: changesets/action@v1
         with:
+          commit: 'chore: release browserify-util'
+          title: 'chore: release browserify-util'
+          version: npx @changesets/cli version
           publish: npx @changesets/cli publish
```

Comment:
This is the first attempt to add `changesets` auto publish ([`3ca5321` (#3)](https://github.com/un-es/node-util/pull/3/commits/3ca532172b7816f2a11071b4f1cfe0efdeb8890b)).

I didn't use changesets CI auto-publish feature before, would you mind checking if this is correct?

Reply:
They seems correct.

I just found I had a typo at `npx cnpm sync browserify`, would you mind to change it to `npx cnpm sync browserify-util`?